typings/azdataGraph.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'azdataGraph/dist/build' {
```

Comment:
@kburtram , I will see if we can move some typing to the control. That should fix some of the import problems.

Reply:
thanks!